contrib/etckeeper/template.py

```diff
@@ -0,0 +1,19 @@
+pkgname = "etckeeper"
+pkgver = "1.18.21"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+make_install_args = [
+    "PYTHON=/bin/false",
+    "zshcompletiondir=${prefix}/share/zsh/site-functions",
+]
+make_check_target = "test"
+hostmakedepends = ["gmake", "gsed", "perl"]
```

Comment:
where is the gsed being used?

Reply:
I thought it was needed [here in the makefile](https://git.joeyh.name/index.cgi/etckeeper.git/tree/Makefile#n91), but it indeed works without both gsed and perl.

I've pushed the fix removing these two dependencies